core/pages/Product.js

```diff
@@ -115,10 +113,10 @@ export default {
       return this.$store.state['wishlist'] ? this.$store.dispatch('wishlist/removeItem', product) : false
     },
     addToList (list) {
-      return this.$store.state[list] ? this.$store.dispatch(`${list}/addItem`, this.product) : false
+      CompareProduct.methods.addToCompare.call(this, this.product)
```

Comment:
dont forget about comments what has changed in this method. they'll be used for migrations guide

Reply:
👍